src/test/java/org/pih/petl/job/PetlScheduledExecutionTaskTest.java

```diff
@@ -23,6 +24,7 @@
 
 @RunWith(SpringRunner.class)
 @SpringBootTest
+@TestPropertySource(properties = {"petl.jobDir = src/test/resources/configuration/jobs/scheduler"})
```

Comment:
I needed to make this change in order to get these unit tests to pass when run via maven command line.

Reply:
Looks like a better way to do it than how I did it anyway!